lib/updatehub/updatehub.c

```diff
@@ -66,6 +66,8 @@ static struct update_info {
 	int image_size;
 } update_info;
 
+static struct k_delayed_work updatehun_work_handle;
```

Comment:
Looks like a typo, perhaps this should be *updatehu**b**_work_handle*

Reply:
Ops! Thank you!